### PR TITLE
feat(Issue 4): Visualizzare lo spettrogramma delle registrazioni

### LINKED ISSUE #4: Visualizzare lo spettrogramma delle registrazioni (convolutivi)
Molto simile al task precedente (#2), in questo caso ci accertiamo di mostrare le differenze visive tra i dati che daremo in input ai vari modelli scelti. In questo caso i modelli di questo specifico task sono:
- convolutivo

Quindi mostriamo lo spettrogramma, e commentiamo le differenze visive (occhio nudo, o se riuscite anche a livello di codice o in altro modo) tra gli spettrogrammi di diverse label.

Questa task è stata "creata" da @HiImGabry, per maggiori informazioni quindi contattate lui.

### PR DESCRIPTION
**Pull request by HiImGabry**
Close #4

### CODE REVIEW

**Comment by HiImGabry** [comment]
Non so il perché ma la frase "...Infine restituisce questi valori, da quale file audio sono stati estratti e in quale cartella era presente quel file..." non mi convince a livello di Italiano corretto.

**Comment by HiImGabry** [comment]
Ti ho aggiunto la cella di installazione delle librerie che hai usato, poi decideremo se mantenerla o no. Inoltre ho cambiato la frase dopo il paragrafo "Importazione delle Librerie Necessarie" in da "Per prima cosa, importiamo tutte le librerie necessarie" a "Per prima cosa, installiamo e importiamo tutte le librerie necessarie".

**Comment by HiImGabry** [comment]
La roba che hai fatto va bene e funziona, però vorrei che invece di utilizzare os e le directory, utilizzassimo direttamente il dataset. Quindi dovremo fare "train_ds = tf.keras.utils.audio_dataset_from_directory(directory='../reduced_dataset/train/audio')" e poi prenderci quello che ci serve.

Se usiamo l'os-dir invece dobbiamo per forza avere tutti lo stesso setup di progetto, che se volete si può fare. Fammi sapere se hai capito e cosa ne pensi
